tests/priority_queue_test.c

```diff
@@ -103,7 +103,7 @@ static int test_priority_queue_random_values(struct aws_allocator *alloc, void *
     int storage[SIZE], err;
     aws_priority_queue_static_init(&queue, storage, SIZE, sizeof(int), compare_ints);
     int values[SIZE];
-    srand((unsigned)&queue);
+    srand((size_t)&queue);
```

Comment:
I don't think this is right on 64 bit systems. It should probably be:

size_t queue_addr = (size_t)&queue;
srand((unsigned)queue_addr);